code-challenges.js

```diff
@@ -4,7 +4,16 @@
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+ const fibo = () => {
+    let arr=[]
+    arr[0]=0
+    arr[1]=1
+    for(i=2; i<=10; i++)
+    { arr[i]=arr[i-1]+arr[i-2] }
+    return arr
+}
 
+console.log("Fibonacci:", fibo())
 
```

Comment:
This is a challenging problem. I like your approach but I think you have a couple extra lines of code. It looks like you are creating an array then adding two values to the array on lines 9 and 10. I think you could shorten that up by just giving your array two values `let arr=[0, 1]`